### PR TITLE
Create boxy.rb

### PR DESCRIPTION
**Pull request by AndrewSB**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by AndrewSB** [comment]
currently auditing and testing this new Cask

File: Casks/boxy.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'boxy' do
+  version '1.1'
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/boxy.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/boxy.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url 'https://boxyapp.s3.amazonaws.com/trials/1.1/Boxy.zip'
```
Comment (vitorgalvao) [inline Casks/boxy.rb]: `url "https://boxyapp.s3.amazonaws.com/trials/#{version}/Boxy.zip"`

Comment (vitorgalvao) [inline Casks/boxy.rb]: Needs a `# boxyapp.s3.amazonaws.com was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by vitorgalvao** [comment]
https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#trial-and-freemium-versions

**Comment by AndrewSB** [comment]
I see, thank you for the feedback @vitorgalvao. Will reattempt this PR only if the full version is made available without visiting the mac app store

**Pull request closed without merging**